### DISCUSSION TITLE
Automated SMTP server configuration?

### DISCUSSION THREAD

**Discussion by acdha** [DependencyTrack/dependency-track#3187] [Q&A]
It seems like an odd gap that the SMTP settings can be configured through the admin area but not using the config file or environmental variables. Did I miss something or is there a deeper reason why that isn't offered? In my hosting environment (AWS ECS) it would be easy to have each of those pulled from the platform secrets store so we wouldn't have to have humans in the loop for tasks like password rotation.

**Comment by Monishguru** [comment]
Am trying to deploy dependency track as docker container.Is it possible to enable email and  configure the SMTP details via Chef.Do we have the config environment variables. I don't see such documentation to enable email  and configuration via environment variables of the docker container.

I found few variables as below 

ALPINE_MAIL_ENABLED=TRUE 

but using this never enabled the email in dependency track

**Comment by Monishguru** [comment]
https://github.com/DependencyTrack/dependency-track/issues/2569#issuecomment-3188905277 

According to this comment its not possible as of now to configure smtp in an automated way
